compilation failing "Can't resolve 'next-server/dynamic' "

Unable to compile, throwing error: -
Failed to compile
./components/EditorPanel.tsx
Module not found: Can't resolve 'next-server/dynamic' in '/home/prasham/Desktop/transform/components'
This error occurred during the build time and cannot be dismissed.

![dynamic-error](https://user-images.githubusercontent.com/13845070/67163278-ac1b0880-f38a-11e9-95e5-089bea64950f.png)

--- a/components/EditorPanel.tsx
+++ b/components/EditorPanel.tsx
@@ -14,7 +14,7 @@ import {
   Tooltip
 } from "evergreen-ui";
 import React, { useCallback, useEffect, useRef, useState } from "react";
-import dynamic from "next-server/dynamic";
+import dynamic from "next/dynamic";
 import copy from "clipboard-copy";
 import { getWorker, Wrapper } from "@utils/workerWrapper";
 import Npm from "@assets/svgs/Npm";

